From 36a7502efc5c59e18fba91ed0d33b44435102ed0 Mon Sep 17 00:00:00 2001
From: Eric Anholt <eric@anholt.net>
Date: Wed, 21 Jun 2017 09:03:51 -0700
Subject: [PATCH 172/215] bcm2708: Drop CMA alignment from FKMS mode as well.

I dropped it from KMS mode in d88274d88ed81de1ade8e18e4c0ed91792ec82ea
and should have done both of them at that time.

Signed-off-by: Eric Anholt <eric@anholt.net>
---
 arch/arm/boot/dts/overlays/vc4-fkms-v3d-overlay.dts | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/arch/arm/boot/dts/overlays/vc4-fkms-v3d-overlay.dts b/arch/arm/boot/dts/overlays/vc4-fkms-v3d-overlay.dts
index 95a595a..36fbf6c 100644
--- a/arch/arm/boot/dts/overlays/vc4-fkms-v3d-overlay.dts
+++ b/arch/arm/boot/dts/overlays/vc4-fkms-v3d-overlay.dts
@@ -11,35 +11,35 @@
 	fragment@0 {
 		target-path = "/chosen";
 		__overlay__ {
-			bootargs = "cma=256M@256M";
+			bootargs = "cma=256M";
 		};
 	};
 
 	fragment@1 {
 		target-path = "/chosen";
 		__dormant__ {
-			bootargs = "cma=192M@256M";
+			bootargs = "cma=192M";
 		};
 	};
 
 	fragment@2 {
 		target-path = "/chosen";
 		__dormant__ {
-			bootargs = "cma=128M@128M";
+			bootargs = "cma=128M";
 		};
 	};
 
 	fragment@3 {
 		target-path = "/chosen";
 		__dormant__ {
-			bootargs = "cma=96M@128M";
+			bootargs = "cma=96M";
 		};
 	};
 
 	fragment@4 {
 		target-path = "/chosen";
 		__dormant__ {
-			bootargs = "cma=64M@64M";
+			bootargs = "cma=64M";
 		};
 	};
 
-- 
1.9.1

